src/actions/analytics.ts

```diff
@@ -137,14 +125,8 @@ export function AnalyticsActions<ActionsBase extends Constructor<TinlakeParams>>
       return (await this.contract('JUNIOR_TOKEN').totalSupply()).toBN()
     }
 
-    // REV: remove
     getMaxSupplyAmountJunior = async (user: string) => {
-      return (await this.contract('JUNIOR_OPERATOR').maxCurrency(user)).toBN()
-    }
-
-    // REV: remove
-    getMaxRedeemAmountJunior = async (user: string) => {
-      return (await this.contract('JUNIOR_OPERATOR').maxToken(user)).toBN()
+      return (await this.contract('TINLAKE_CURRENCY').balanceOf(user)).toBN()
```

Comment:
Do we really need this? It is a duplicate of `getCurrencyBalance` in `src/actions/currency.ts`.

Reply:
ah, didn't check currency. i can remove this function